Assets/Code/src/Runtime/Match/Components/MatchCountdownUI.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+namespace HouraiTeahouse.FantasyCrescendo.Matches
+{
+
+public class MatchCountdownUI : MonoBehaviour
+{
+    // Component References
+    // EditorBrowsable attribute hides the public variables from intellisense
+    // In other words, you can still access it, but it won't be in the autocomplete list
+    // If variables are null, look for them
+    [Header("Component References")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public TextMeshProUGUI _TextUI;
```

Comment:
Make this private, and annotate with `UnityEngine.SerializeField` instead. Same with below.

If it's private: `_textUI` vs `_TextUI`.